src/commands/interviews/interviewers.ts

```diff
@@ -0,0 +1,61 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import _ from 'lodash';
+
+import { BaseCommand } from '../../utils/commands';
+import { availableDomains, getAvailableDomainsString, getInterviewers, Interviewer } from '../../components/interview';
+import { EMBED_COLOUR } from '../../utils/embeds';
+import { parseDomainArg, validateDomainArg } from './utils';
+
+const RESULTS_PER_PAGE = 6;
+
+class InterviewersCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'interviewers',
+      aliases: ['interviewer', 'interviewer-list', 'interviewers-list'],
+      group: 'interviews',
+      memberName: 'interviewers',
+      args: [
+        {
+          key: 'domain',
+          prompt: `enter one of ${getAvailableDomainsString()}.`,
+          type: 'string',
+          default: '',
+          validate: validateDomainArg,
+          parse: parseDomainArg
+        }
+      ],
+      description: 'Shows you a list of available interviewers.',
+      examples: ['.interviewers', '.interviewers backend']
+    });
+  }
+
+  private async getInterviewerDisplayInfo(interviewer: Interviewer) {
+    const userTag = (await this.client.users.fetch(interviewer['user_id'])).tag;
+    return '**' + userTag + '** | [Calendar](' + interviewer['link'] + ')\n\n';
+  }
+
+  async onRun(message: CommandoMessage, args: { domain: string }): Promise<Message> {
+    const { domain } = args;
+
+    // query interviewers
+    const interviewers = await getInterviewers(args.domain);
+    // shuffles interviewers to load balance
+    const shuffledInterviewers = _.shuffle(interviewers);
+    // only show up to page limit
```

Comment:
Is this part going to be an issue for when we want to move from page 1 to page 2 (with like the new buttons or something? We don't really have page states of what we've already seen after getting the shuffled interviewers. Not sure if that's just intended basic feature rn (which is fine for now I guess, but we will need to think about shuffling and then paginating)

Also, let me know if you don't understand me LOL I don't think I'm explaining this well.

Reply:
I thought we decided to not pagination? Pagination with shuffling is much more difficult and is definitely outside the scope of this PR. We should either choose shuffling or pagination for now.